diffrax/brownian/tree.py

```diff
@@ -70,10 +124,12 @@ def __init__(
         tol: Scalar,
         shape: Union[Tuple[int, ...], PyTree[jax.ShapeDtypeStruct]],
         key: "jax.random.PRNGKey",
+        spacetime_levyarea: bool = False,
     ):
-        self.t0 = t0
-        self.t1 = t1
-        self.tol = tol
+        self.t0 = jnp.minimum(t0, t1)
+        self.t1 = jnp.maximum(t0, t1)
+        self.tol = tol / (self.t1 - self.t0)
```

Comment:
This is a change from previous behaviour?

Reply:
Yes, because of normalising the interval. This ensures that the behaviour is the same in fact, but avoids computing tol/(t1-t0) each time in the main tree loop.

Reply:
Yep.